source/DialogueBox.hx

```diff
@@ -240,14 +240,14 @@ class DialogueBox extends FlxSpriteGroup
 
 		switch (curCharacter)
 		{
-			case 'dad':
+			case 'opponent':
 				portraitRight.visible = false;
 				if (!portraitLeft.visible)
 				{
 					portraitLeft.visible = true;
 					portraitLeft.animation.play('enter');
 				}
-			case 'bf':
+			case 'player':
```

Comment:
i do think there should be some backwards compatibility here, mainly because there's many mods that change the dialogue

can prob just do `case 'opponent' | 'dad'` and `case 'player' | 'bf'`